tests/acceptance/test_organization_security_privacy.py

```diff
@@ -56,3 +58,52 @@ def test_setting_2fa_without_2fa_enabled(self):
         self.browser.wait_until_not(".modal")
         self.browser.wait_until_test_id("toast-error")
         self.load_organization_helper("setting 2fa without 2fa enabled")
+
+    def test_renders_advanced_data_scrubbing_without_rule(self):
+        user_owner = self.create_user("owner@example.com")
+        organization = self.create_organization(name="Example", owner=user_owner)
+        self.login_as(user_owner)
```

Comment:
I think each test should have an owner + org (and project) created, you can access it with `self.user` and `self.org`

Reply:
Yes, the org and user are created the first time you use those properties. The same is true for `self.project`